sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExpression.scala

```diff
@@ -204,8 +204,6 @@ case class GpuWindowExpression(windowFunction: Expression, windowSpec: GpuWindow
       case other =>
         throw new IllegalStateException(s"${other.getClass} is not a supported window aggregation")
     }
-    // Add support for Pandas (Python) UDF
-    case pythonFunc: GpuPythonUDF => pythonFunc
```

Comment:
`GpuPythonUDF` derives from `GpuAggregateWindowFunction` which is why the compiler was complaining.  This has the same code as that for the `GpuAggregateWindowFunction` hence it can simply be deleted.